lib/init/grass.py

```diff
@@ -1373,19 +1374,11 @@ def set_language(grass_config_dir):
     encoding = None
 
     # Override value is stored in wxGUI preferences file.
-    # As it's the only thing required, we'll just grep it out.
-    try:
-        fd = open(os.path.join(grass_config_dir, "wx"), "r")
-    except:
-        # Even if there is no override, we still need to set locale.
-        pass
-    else:
-        for line in fd:
-            if re.search("^language", line):
-                line = line.rstrip(" %s" % os.linesep)
-                language = "".join(line.split(";")[-1:])
-                break
-        fd.close()
+    with open(os.path.join(grass_config_dir, "wx.json"), "r") as json_file:
+        try:
+            language = json.load(json_file)["language"]["locale"]["lc_all"]
+        except KeyError:
+            language = None
```

Comment:
+1, more clear than the previous code. I'm not sure if `language == "None"` is still relevant with the new format. I would hope it is not.

Reply:
Lets hope so. I removed this old workaround.